workspaces/theme/source.json

```diff
@@ -1 +1 @@
-{"repo":"https://github.com/redhat-developer/rhdh-plugins","repo-ref":"87dd8016e0e04a8e7c7c228b7fa2c66ff95e2306","repo-flat":false,"repo-backstage-version":"1.42.5"}
```

Comment:
And this ref looks not correct to me

https://github.com/redhat-developer/rhdh-plugins/commit/99f080a13603961cd868165a45eef284d04a30a3